app/views/brexit_checker/_action_list.html.erb

```diff
@@ -5,7 +5,7 @@
             class="govuk-link"
             href="<%= action.title_url %>"
             data-module="track-click"
-            data-track-action="<%= audience_heading %> <%= action_index %> - Action"
+            data-track-action="<%= "#{analytics_group}#{index + 1}"%> - Action"
```

Comment:
We will need to let PAs know about this change

Reply:
Yeah wanted to check in on this, wasn't sure if it's almost instantly superseded by the additional tracking work?

Reply:
@huwd The ecommerce tracking? I think for now at least, we're having the two running alongside each other, but it's probably worth checking.

Reply:
Cool, would be good to take 5 and give this a good check tomorrow!

Reply:
Ok that's up! Will resolve these and we can check in on the heroku app